composer.json

```diff
@@ -86,7 +86,8 @@
         "monolog/monolog": "~1.11",
         "ocramius/proxy-manager": "~0.4|~1.0|~2.0",
         "egulias/email-validator": "~1.2",
-        "phpdocumentor/reflection": "^1.0.7"
+        "phpdocumentor/reflection": "^1.0.7",
+        "twig/twig": "~1.23|~2.0"
```

Comment:
we already have Twig as a regular requirement. Having it in both section is bad as it can create weird behaviors in Composer (unlikely here for ow as the constraint is the same, but they may likely get out of sync)